packages/client/hmi-client/src/services/ClientEventService.ts

```diff
@@ -84,14 +82,10 @@ export async function init(): Promise<void> {
  * and reconnects if not
  */
 setInterval(async () => {
-	if (!reconnecting) {
-		const config = await getConfiguration();
-		const heartbeatIntervalMillis = config?.sseHeartbeatIntervalMillis ?? 10000;
-		if (new Date().valueOf() - lastHeartbeat > heartbeatIntervalMillis) {
-			reconnecting = true;
-			await init();
-			reconnecting = false;
-		}
+	const config = await getConfiguration();
```

Comment:
Without `reconnecting` here, don't we run the risk of calling `init()` while we're still waiting on a previous call to `init()`?